pullrequest.rst

```diff
@@ -47,6 +47,10 @@ Here is a quick overview of how you can contribute to CPython:
 .. [*] If an issue is trivial (e.g. typo fixes), or if an issue already exists,
        you can skip this step.
 
+.. note::
+   Please keep the commit history in the pull request intact by not squashing,
+   amending, or anything that would require a force push to GitHub.
+
```

Comment:
Rather than being an exact copy of the sentence in [section 3.8](https://devguide.python.org/pullrequest/#submitting), it would be better to make the note a summarized and rephrased version of it. This section should aim to give a very brief overview, and in general, we definitely want to avoid repeating the same sentence.

E.g. "In order to keep the commit history intact, please avoid force-pushing to the PR"

Reply:
That is a good point that I did not see. The shorter note you suggested is easier to read - thanks!